api/v1alpha1/nnfstorageprofile_webhook.go

```diff
@@ -99,7 +99,10 @@ func (r *NnfStorageProfile) validateContent() error {
 
 func (r *NnfStorageProfile) validateContentLustre() error {
 	if r.Data.LustreStorage.CombinedMGTMDT && len(r.Data.LustreStorage.ExternalMGS) > 0 {
-		return fmt.Errorf("Cannot set both combinedMgtMdt and externalMgs")
+		return fmt.Errorf("cannot set both combinedMgtMdt and externalMgs")
+	}
+	if r.Data.LustreStorage.CombinedMGTMDT && r.Data.LustreStorage.ExclusiveMDT {
+		return fmt.Errorf("cannot set both combinedMgtMdt and exclusiveMdt; the target will already be constrained because it is an MGT")
```

Comment:
I don't think this is true. The MGT will be prevented from colocating with other MGTs, but it could shared a node with an OST.

Reply:
Sure.  I'll remove that constraint.